rocketpy/Environment.py

```diff
@@ -3329,7 +3330,7 @@ def allInfoReturned(self):
 
     def exportEnvironment(self, filename="environment"):
         """Export important attributes of Environment class so it can be used
-        again in further siulations by using the customAtmosphere atmospheric
+        again in further simulations by using the customAtmosphere atmospheric
         model.
```

Comment:
"""
Export important attributes of Environment class to a .json file, saving all the information needed to recreate the same environment using customAtmosphere.
"""

Would be a better description here, might as well change it now.

Reply:
Fixed